qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py

```diff
@@ -21,6 +21,7 @@ def upgrade():
     op.add_column('collection', sa.Column('description', sa.String(), nullable=True))
     op.add_column('collection', sa.Column('owner', sa.String(length=100), nullable=True))
     op.add_column('collection', sa.Column('view_url', sa.String(), nullable=True))
+    op.add_column('collection', sa.Column('view_metadata', sa.JSON(), nullable=True))
```

Comment:
@dgasmith Could you please spot-review this two-line change to the migration?

Reply:
Looks good.